components/Highlights.tsx

```diff
@@ -65,8 +65,8 @@ export function Highlights({ features }) {
       </Box>
 
       <Box css={{ width: 'fit-content' }} as="nav" aria-labelledby="site-component-info-heading">
-        <VisuallyHidden as="h2" id="site-component-info-heading">
-          Component Reference Links
+        <VisuallyHidden asChild id="site-component-info-heading">
+          <h2>Component Reference Links</h2>
```

Comment:
I know this works technically but I feel like DOM attributes sit better on the element itself then

```suggestion
        <VisuallyHidden asChild>
          <h2 id="site-component-info-heading">Component Reference Links</h2>
```